model/src/pyrenew/process/periodiceffect.py

```diff
@@ -110,9 +114,11 @@ def sample(self, duration: int, **kwargs):
         """
 
         return PeriodicEffectSample(
-            value=self.broadcaster(
-                data=self.quantity_to_broadcast.sample(**kwargs)[0],
-                n_timepoints=duration,
+            value=SampledValue(
```

Comment:
Should this return a `SampledValue`, not an `PeriodicEffectSample`?

Reply:
Actually, I think the plan was to return a regular tuple, and later have a PR to convert everything to named tuples.

Reply:
PeriodicEffectSample is (was) a namedtuple before this PR. So there's no need to change here.

Reply:
I would prefer to remove `PeriodicEffectSample` and just return a namedtuple without a special class.